### ISSUE TITLE
How can i reload or what is the right way to get the react-native-video player to show/hide subtitles

### ISSUE THREAD

**Issue by cvaldezscse** [TheWidlarzGroup/react-native-video#1431]
### Current behavior
I need to toggle subtitles on/off, but the lifecycle of the application is not doing it. Right now the subtitles. Here i need to know how can i do it, i added an extra button to toggle subtitles in the app. Do you know how to do it?

### Reproduction steps
Implement like the documentation of this repo says

### Expected behavior
toggle subtitles On and Off

### Platform
Which player are you experiencing the problem on:
* iOS

### Video sample
If possible, include a link to the video that has the problem that can be streamed or downloaded from.

**Comment by alikazemkhanloo** [comment]
to show/hide subtitles you can do as the [documentation](https://github.com/react-native-community/react-native-video#selectedtexttrack) says:
```Javascript
selectedTextTrack={{
  type: this.state.subtitle_off ? "disabled" : "title", 
  value: "English Subtitles"
}}
```
and change the state when you want to hide/show.

**Comment by Landerson352** [comment]
I have the above solution implemented in my app, but when I disabled subtitles on iOS, the last one to display remains on the screen. No new words appear, but the words that were visible when I disabled subtitles stay visible on the screen for the remainder of the video duration. Any ideas?

Edit:

I am seeing this issue referenced in a few places:
- https://github.com/react-native-community/react-native-video/issues/1144
- https://github.com/react-native-community/react-native-video/pull/1468

Doesn't seem solved yet, but I hope this helps someone.
